apiserver/facades/client/cloud/instance_information.go

```diff
@@ -63,8 +73,9 @@ func instanceTypes(api *CloudAPI,
 			value = *cons.Constraints
 		}
 		backend := cloudEnvironConfigGetter{
-			Backend: api.backend,
-			region:  cons.CloudRegion,
+			Backend:        api.backend,
+			region:         cons.CloudRegion,
```

Comment:
I can't see where region is used. We can probaly drop the use of `cloudEnvironConfigGetter` struct and use the new helper added in this PR